ui/src/helpers/get-status-class.ts

```diff
@@ -149,3 +150,73 @@ export function getRuleViolationSeverityBackgroundColor(
 export function getIssueSeverityBackgroundColor(severity: Severity): string {
   return ISSUE_SEVERITY_BG_COLOR[severity];
 }
+
+// Get the color class for coloring the background of ecosystems.
+// These color classes are defined as follows:
+// 1. To avoid clashing with "status indicator colors" (eg. red for "FAILED")
+//    elsewhere in the UI, those TailwindCSS color palettes which are already
+//    used for status indicators are excluded, which leaves 15 different color
+//    palettes to choose from.
+// 2. The package managers are grouped according to the programming languages,
+//    with a total of 14 different groups.
+// 3. Package managers which belong to the same group are chosen from inside
+//    these 14 color palettes in a way that the colors are visually distinct.
+//
+// TODO: This probably needs to be revisited later on, when we see overlaps in
+//       the colors used for the package managers.
```

Comment:
I think this comment can be removed, as it will be apparent from using the UI if this needs to be revisited.